sentry-okhttp/src/main/java/io/sentry/okhttp/SentryOkHttpEvent.kt

```diff
@@ -150,6 +153,8 @@ internal class SentryOkHttpEvent(private val hub: IHub, private val request: Req
         hint.set(TypeCheckHint.OKHTTP_REQUEST, request)
         response?.let { hint.set(TypeCheckHint.OKHTTP_RESPONSE, it) }
 
+        // needs this as unix timestamp for rrweb
+        breadcrumb.setData("end_timestamp", CurrentDateProvider.getInstance().currentTimeMillis)
```

Comment:
Maybe makes sense to have `"start_timestamp"` and `"end_timestamp"` as constants, similar to `SpanDataConvention`?